test/activities/quizzes/data/NoneditableQuiz.js

```diff
@@ -20,13 +20,6 @@ export const nonEditableQuiz = {
 			'href': 'http://d2l-d6xpge1xiym.desire2learn.d2l:44444/d2l/lms/quizzing/quizzing.d2l?ou=6606&qi=22',
 			'type': 'text/html'
 		},
-		{
```

Comment:
Removed this from the non editable quiz to write tests for the case that a user can't preview a quiz